lib/Drush/CommandFiles/core/DrupliconCommands.php

```diff
@@ -0,0 +1,42 @@
+<?php
+namespace Drush\CommandFiles\core;
+
+use Consolidation\AnnotatedCommand\CommandData;
+
+class DrupliconCommands {
+  protected $printed = false;
+
+  /**
+   * Print druplicon as post-command output.
+   *
+   * @hook post-command *
+   * @option $druplicon Shows the druplicon as glorious ASCII art.
```

Comment:
Oh, I fixed that in the upstream -- the $ is now optional.  I'll push a new commit to remove it.